realm/realm-library/src/androidTest/java/io/realm/rule/TestRealmConfigurationFactory.java

```diff
@@ -151,4 +152,17 @@ public void copyRealmFromAssets(Context context, String realmPath, String newNam
         outputStream.close();
         is.close();
     }
+
+    public static void emulateRxJavaUnavailable(RealmConfiguration config) {
+        //noinspection TryWithIdenticalCatches
+        try {
+            final Field field = config.getClass().getDeclaredField("rxObservableFactory");
```

Comment:
What about set `RealmConfiguration.rxJavaAvailable` to `new Boolean(false)`? current implementation is also fine to me.